caom2utils/caom2utils/parsers.py

```diff
@@ -1871,15 +1886,28 @@ def _get_num_parts(self):
             result = 1
         return result
 
+    def _set_wcs_parsers(self, obs_blueprint):
+        # self._wcs_parsers[0] = WcsParser(obs_blueprint, extension=0)
+        # used to set the astropy wcs info, resulting in a validated wcs
+        # that can be used to construct a valid CAOM2 record
+        self._wcs_parsers = {}
```

Comment:
Argument not required. In fact I'm not sure the entire method is required.

Reply:
This method call is over-writing the default behaviour in the `ContentParser` class. The default behaviour does require access to the `obs_blueprint`.

Reply:
Maybe add a comment so we (I) remember next time. Too bad Python doesn't have the `@override` decorator like Java has.

Reply:
Added.